tests/ptvsd/server/test_multiproc.py

```diff
@@ -15,130 +15,123 @@
 from tests.timeline import Event, Request
 
 
-pytestmark = pytest.mark.skip("https://github.com/microsoft/ptvsd/issues/1706")
+# pytestmark = pytest.mark.skip("https://github.com/microsoft/ptvsd/issues/1706")
 
 
 @pytest.mark.timeout(30)
-@pytest.mark.skipif(
-    platform.system() != "Windows",
-    reason="Debugging multiprocessing module only works on Windows",
-)
 @pytest.mark.parametrize(
-    "start_method", [runners.launch, runners.attach_by_socket["cli"]]
+    "start_method",
+    [""]
+    if sys.version_info < (3,)
+    else ["spawn"]
+    if platform.platform == "Windows"
```

Comment:
`platform.platform` is actually a callable (i.e.: platform.platform() returns `'Windows-10-10.0.17763-SP0'` to me).

I think you could use `sys.platform == 'win32'` instead to check if it's windows...

Reply:
This was meant to be `platform.system()`, which is generally what we use in the tests for OS checks.

Reply:
Well, that's not wrong per-se, but usually I'd advise to use `sys.platform` which is set at compile-time.

From what I remember `platform.system()` does end up spawning a process to get its info... just as an anecdote, I had an issue once -- several years ago -- where that call halted the process startup... I think it was back in Python 2.4, so, it may not be an issue anymore, but just the fact that it needs to spawn a process to get that info which should usually be as good as `sys.platform` made me always default to `sys.platform` -- see: https://github.com/microsoft/ptvsd/blob/master/src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_constants.py#L73 for how `pydevd` defines the related constants to know the OS.

Reply:
Interesting, I didn't know that! We should look into it, but given the pervasive use of `platform` in the tests, and occasionally in the adapter, it should be a separate issue to make sure we change them all.

Reply:
https://github.com/microsoft/ptvsd/issues/1884